simpinkscr/simple_inkscape_scripting.py

```diff
@@ -3724,7 +3724,8 @@ def effect(self):
 
         # Remove an unnecessary import that may be introduced when
         # running from Visual Studio Code.
-        code.replace("from simpinkscr import *", "")
+        pattern = r"^(import (inkex|simpinkscr)|(from (inkex|simpinkscr) import.*))\r*\n*"
```

Comment:
Ah yes.  Good call.  I have a new regex I'm about to update it to that I believe handles every case.  I had to adjust the beginning to just look for tab and space because \s matches line feeds so it was pulling blank lines prior to the import line.